src/eventSchemas/models/schemaCodeLangs.ts

```diff
@@ -9,9 +9,10 @@ import { Set as ImmutableSet } from 'immutable'
 export const JAVA = 'Java 8+'
 export const PYTHON = 'Python 3.6+'
 export const TYPESCRIPT = 'Typescript 3+'
-export type SchemaCodeLangs = 'Java 8+' | 'Python 3.6+' | 'Typescript 3+'
+export const GOLANG = 'Go 1+'
```

Comment:
Does this (and the other consts) need to be exported? Mainly calling it out because these const names are pretty generic and might get confusing since these are only used for schemas.